src/main/java/com/exactpro/th2/crawler/processor/healer/grpc/HealerImpl.java

```diff
@@ -232,6 +232,7 @@ private void heal(Collection<EventData> events) throws IOException, InterruptedE
                         return eventAncestors;
                     }
                 }
+                else notFoundParent.remove(parentId);
```

Comment:
```suggestion
                } else {
                    notFoundParent.remove(parentId);
                }
```
Set operator brackets for if, else operators